modules/stanford_layout_paragraphs/scss/preview_paragraphs.scss

```diff
@@ -0,0 +1,45 @@
+
+#edit-su-page-components {
```

Comment:
Use a class because these styles will be applied to all content types, not just the basic page.

Reply:
Can we add a class that only shows on that preview?